### PR TITLE
docs: reflect latest features

### PR DESCRIPTION
**Pull request by matmen**
Implements docs changes for:

- [x] #425 
- [x] #486 
- [x] #496 (only screenshot change)
- [x] #452 (only screenshot change)
- [x] #500 (only screenshot change)
- [x] update [landing page screenshot](https://docs.fluidd.xyz/assets/images/preview_sliced.png)
- [x] update [readme screenshot](https://raw.githubusercontent.com/fluidd-core/fluidd/develop/.github/images/preview_sliced.png)

### CODE REVIEW

File: docs/features/cameras.md [hunk 1]
```diff
@@ -26,21 +26,25 @@ You can add up to four cameras to display on your dashboard.
 
 Currently supported types are;
 
-- MJPEG Stream
+- MJPEG Stream
```
Comment (pedrolamas) [inline docs/features/cameras.md:29]:
```suggestion
- MJPEG Stream
```

File: docs/features/cameras.md [hunk 2]
```diff
@@ -26,21 +26,25 @@ You can add up to four cameras to display on your dashboard.
...
 Currently supported types are;
 
-- MJPEG Stream
+- MJPEG Stream  
   This is the traditional mjpegstream service. The service pushes images to
   fluidd at the configured resolution and FPS you have setup. This requires
   a lot of bandwidth, and can cause issues with unstable network connections.
 
-- MJPEG Adaptive
+- MJPEG Adaptive
```
Comment (pedrolamas) [inline docs/features/cameras.md:34]:
```suggestion
- MJPEG Adaptive
```

File: docs/features/cameras.md [hunk 3]
```diff
@@ -26,21 +26,25 @@ You can add up to four cameras to display on your dashboard.
...
   a lot of bandwidth, and can cause issues with unstable network connections.
 
-- MJPEG Adaptive
+- MJPEG Adaptive  
   This will PULL images from the mjpegstream service, using the snapshot URL.
   You can set a target FPS, and your browser will try to keep up with the
   intended target. This can be a more reliable approach for some.
 
-- IP Camera
+- IP Camera
```
Comment (pedrolamas) [inline docs/features/cameras.md:39]:
```suggestion
- IP Camera
```

File: docs/features/cameras.md [hunk 4]
```diff
@@ -26,21 +26,25 @@ You can add up to four cameras to display on your dashboard.
...
   You can set a target FPS, and your browser will try to keep up with the
   intended target. This can be a more reliable approach for some.
 
-- IP Camera
+- IP Camera  
   This is an experimental option. Effectively, it swaps out the `<img>` tag
   for a `<video>` tag. You should only use this if you can provide a URL
   that supports native HTML5 video tags.
 
+- HTTP Page
```
Comment (pedrolamas) [inline docs/features/cameras.md:44]:
```suggestion
- HTTP Page
```

File: docs/features/cameras.md (hunk 1 above)
Comment (matmen) [reply docs/features/cameras.md:29]: these changes are intentional, they improve the layout (making `MJPEG Stream` a "heading" in this case)
without the two spaces there wouldn't be a linebreak
i don't think it's worth opening another PR for these small changes

Comment (pedrolamas) [reply docs/features/cameras.md:29]: Fair enough, that makes total sense! 🙂

**Comment by matmen** [comment]
@max-scopp @randellhodges Please check if the documentation for your contributions are sufficient and let me know if you need something changed

**Review by pedrolamas — APPROVED** [review]
LGTM

**Review by pedrolamas — APPROVED** [review]

**Pull request merged**